CMakeLists.txt

```diff
@@ -73,10 +73,14 @@ if(BUILD_LIBOSRM)
   add_custom_command(OUTPUT ${BINDING_DIR}/osrm-datastore
                      COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:osrm-datastore> ${BINDING_DIR}
                      DEPENDS osrm-datastore ${BINDING_DIR})
+  add_custom_command(OUTPUT ${BINDING_DIR}/osrm-components
```

Comment:
Ah, thanks! Thinking of it, I see no reason why we don't build the components tool by default now that we no longer depend on gdal. I will remove the `BUILD_COMPONENTS` option from the backend directly.